crates/cheatnet/src/cheatcodes/deploy.rs

```diff
@@ -77,6 +68,23 @@ impl CheatnetState {
                     ))))
                 }),
             CallContractOutput::Panic { panic_data } => {
+                let panic_data_str = panic_data
+                    .iter()
+                    .map(|x| as_cairo_short_string(x).unwrap())
+                    .collect::<Vec<String>>()
+                    .join("\n");
+
+                for invalid_calldata_msg in [
+                    "Failed to deserialize param #",
+                    "Input too long for arguments",
```

Comment:
I think there was one more (in the case of nonexistent constructor)

Reply:
yes, but it has occurred that `CallContractOutput::Error` is returned from `call_contract`, not `CallContractOutput::Panic`